go.mod

```diff
@@ -18,6 +18,7 @@ require (
 )
 
 require (
+	github.com/dave/dst v0.27.3 // indirect
```

Comment:
We need to be very consciouss about the deps added. 
I do not think that we can move within